README.md

```diff
@@ -66,7 +67,7 @@ A valid (read: that follows this interface) IPFS core implementation, must expos
 
 `callback` must follow `function (err, node) {}` signature, where `err` is an error if the operation was not successful and `node` is a MerkleDAG node of the type [DAGNode][]
 
-If no `callback` is passed, a promise is returned.
+Returns a [promise][].
```

Comment:
@dignifiedquire I think I agree about it being undesirable to always return a promise. A counterargument though might be that *lots* of people are already depending on promisify and us deviating from "the norm" may be detrimental to the larger community. Hm--I'm not sure. ping @diasdavid @nginnever too :)